src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -401,6 +402,32 @@ private ProfessionalUser validateInviteUserRequestAndCreateNewUserObject(
         return responseEntity;
     }
 
+    protected void deletePaymentAccountsOfGivenOrganisation(DeletePbaRequest deletePbaRequest,
+                                                            String orgId, String userId) {
+
+        if (isEmpty(deletePbaRequest.getPaymentAccounts())) {
+            throw new InvalidRequest("No PBA number passed in the request");
+        }
+
+        //check if user status is 'ACTIVE'
+        professionalUserService.checkUserStatusIsActiveByUserId(userId);
+
+        Organisation existingOrganisation = organisationService.getOrganisationByOrgIdentifier(orgId);
+
+        //check if organisation is present in the database and that it has payment accounts associated
+        checkOrganisationAndPbaExists(existingOrganisation);
+
+        //if the organisation is present, check if it is 'ACTIVE'
+        organisationIdentifierValidatorImpl.validateOrganisationIsActive(existingOrganisation);
+
+        //check the pba account number format, remove any blank strings passed
+        //And check if pba belongs to the organisation
+        paymentAccountValidator.validatePaymentAccounts(deletePbaRequest.getPaymentAccounts(), orgId);
+
+        //delete the passed pba account numbers from the organisation
+        paymentAccountService.deletePaymentsOfOrganization(deletePbaRequest, existingOrganisation);
```

Comment:
_Non-important suggestion_: can we update the name of this method `deletePaymentsOfOrganization` to use the English spelling of the word `Organisation` to keep it in line with the other method names?